packages/ckeditor5-dev-tests/bin/notify-travis-status.js

```diff
@@ -7,16 +7,22 @@
 
 /* eslint-env node */
 
-/*
-
-This script assumes that is being executed on Travis CI. It requires three environment variables:
-- SLACK_WEBHOOK_URL - a URL where the notification should be sent
-- START_TIME - POSIX time (when the script has begun the job)
-- END_TIME - POSIX time (when the script has finished the job)
-
-In order to enable the debug mode, set the `DEBUG=true` as the environment variable.
-
- */
+// This script assumes that is being executed on Travis CI. It requires three environment variables:
```

Comment:
```suggestion
// This script assumes that is being executed on Travis CI. It requires following environment variables:
```

There are four env variables listed 🙂